<ui version="4.0" >
 <class>Langton</class>
 <widget class="QMainWindow" name="Langton" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>250</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Langton</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="minimumSize" >
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <widget class="QGraphicsView" name="canvas" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainmenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit_menu" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionSetup" />
   </widget>
   <widget class="QMenu" name="menuSimu_menu" >
    <property name="title" >
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionStart" />
    <addaction name="actionStop" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <addaction name="menuMainmenu" />
   <addaction name="menuEdit_menu" />
   <addaction name="menuSimu_menu" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSetup" >
   <property name="text" >
    <string>Setup</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
